### DISCUSSION TITLE
[11 gennaio 2022 - B] Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#199] [Esami]
**Esercizio 1 (8 punti)**

Progettare un circuito sequenziale con un ingresso x due uscite z1 e z0. L’uscita z1 deve essere uguale a 1 se gli ultimi tre bit di ingresso rappresentano un numero negativo dispari, mentre z0 deve essere 1 se gli ultimi 2 bit sono diversi. Si utilizzino FF di tipo SR per il bit di stato più significativo e FF di tipo T per i restanti bit. Non si considerino le sovrapposizioni.

Esempio
```
x  011110010000011
z1 000010000000000
z0 010000010000010
```

**Comment by CasuFrost** [comment]
![Immagine WhatsApp 2023-01-13 ore 12 42 48](https://user-images.githubusercontent.com/97389762/212312691-12fd773e-bcf2-442b-9775-99b2e47093b3.jpg)

**Comment by fraxb04** [comment]
@CasuFrost
è errato il diagramma, allego il mio 
ps: non mi andava di scrivere l'output logic :)
![image](https://user-images.githubusercontent.com/122573871/216678553-8355bf62-ef63-4627-b09c-e8ab2f5b76ab.png)

**Comment by Catins1** [comment]
> @CasuFrost è errato il diagramma, allego il mio ps: non mi andava di scrivere l'output logic :) ![image](https://user-images.githubusercontent.com/122573871/216678553-8355bf62-ef63-4627-b09c-e8ab2f5b76ab.png)

Per quanto riguarda la rappresentazione del circuito non è errato rappresentare F1' e F0' in quanto abbiamo già SR e T?
